### DISCUSSION TITLE
How to update values for each columns of row table from modal?

### DISCUSSION THREAD

**Discussion by nguyentuyen1322** [ant-design/ant-design#42300] [Q&A] [answered]
- data any field modal will display columns of row in table when fill to modal, how will i do ?

**Comment by zombieJ** [comment]
Sorry, I can understand your description. Could you provide a codesandbox about this?

**Reply by nguyentuyen1322** [reply]
each row when clicked will open the modal data when entering from the modal I want to display on the table

**Comment by nguyentuyen1322** [comment, marked as answer]
codesandbox: https://stackblitz.com/edit/react-xezlcb?file=index.js

open modal when i fill value to form i want display value fill to each columns of row table. pls help me

**Reply by nguyentuyen1322** [reply]
@zombieJ
